src/less/workingfiles/submit.less

```diff
@@ -250,17 +251,21 @@ td {
     background-color: @super-light-gray;
 }
 
+
 #queue-table th:first-child {
-    width: 22%;
+
 }
 
+div .table-cell-div{
+    overflow: scroll;
```

Comment:
wasn't sure if you meant remove the scroll bar for aesthetic purposes or the scrolling functionality.  I currently left it so that you can scroll but there's no scroll bar in view.
If this isn't what you meant it shouldn't take too long to change anyway.